### PR TITLE
Add support to containerfile repo to SLE15

### PR DESCRIPTION
**Pull request by gyr**

### CODE REVIEW

**Review by Vogtinator — CHANGES_REQUESTED** [review]
`SUSE:SLE-15-SP4:GA:TEST/containerfile` got renamed to `containers`

For consistency, `sles15-image` should be released into `SUSE:SLE-15-SP4:GA:TEST/containers` as well, that needs to be done explicitly as the other images still go to the default release target.

**Review by Vogtinator — CHANGES_REQUESTED** [review]
Looks like it's missing the part for `Release.Images.To.Publish` as well

File: gocd/sp4.target.gocd.yaml [hunk 1]
```diff
@@ -83,10 +85,16 @@ pipelines:
         - script: |-
             set -e
             PRODUCTS=$(osc -A https://api.suse.de ls SUSE:SLE-15-SP4:GA | grep "^SLES15-SP4" | grep -v release)
-            for product in kiwi-templates-Minimal sles15-image 000product $PRODUCTS; do
+            CONTAINERS=$(osc -A https://api.suse.de results -b -s succeeded SUSE:SLE-15-SP4:GA -r containerfile | cut -f1 -d ' '| uniq)
```
Comment (Vogtinator) [inline gocd/sp4.target.gocd.yaml:88]: I'm a bit suspicious of the `-s succeeded`. Images without that would silently be skipped. There could be a race between the check above and now...

**Comment by vasiliy-ul** [comment]
Hey, ping here. @Vogtinator , do you think this is now good to go?  Can we move it forward?

**Comment by Vogtinator** [comment]
> Hey, ping here. @Vogtinator , do you think this is now good to go? Can we move it forward?

It's on my todo list, I'll have a look later

File: gocd/sp4.target.gocd.yaml [hunk 2]
```diff
@@ -82,11 +84,18 @@ pipelines:
         tasks:
         - script: |-
             set -e
+            osc -A https://api.suse.de release SUSE:SLE-15-SP4:GA -r containerfile
+            osc -A https://api.suse.de release SUSE:SLE-15-SP4:GA sles15-image --target-project=containers
```
Comment (Vogtinator) [inline gocd/sp4.target.gocd.yaml:88]: I think this needs

```suggestion
            osc -A https://api.suse.de release --target-project=SUSE:SLE-15-SP4:GA --target-repository=containers -r images SUSE:SLE-15-SP4:GA sles15-image
```

File: gocd/sp4.target.gocd.yaml [hunk 3]
```diff
@@ -152,8 +169,10 @@ pipelines:
             tasks:
               - script: |-
                   set -e
+                  osc -A https://api.suse.de release SUSE:SLE-15-SP4:GA -r containerfile
+                  osc -A https://api.suse.de release SUSE:SLE-15-SP4:GA sles15-image --target-project=containers
```
Comment (Vogtinator) [inline gocd/sp4.target.gocd.yaml:173]: Same as above

**Review by Vogtinator — CHANGES_REQUESTED** [review]

File: gocd/sp4.target.gocd.yaml (hunk 3 above)
Comment (gyr) [reply gocd/sp4.target.gocd.yaml]: Fixed

**Review by Vogtinator — APPROVED** [review]
Let's see whether it also works in practice

**Comment by suntorytimed** [comment]
Looks like it fails. The to test is now releasing all packages to images. So all of standard is being released to TEST in images.

**Pull request merged**